packages/now-cli/src/commands/alias/set.ts

```diff
@@ -284,34 +185,6 @@ function handleSetupDomainError<T>(
         { extraSpace: '     ' }
       )}\n\n`
     );
-    if (error instanceof ERRORS.DomainVerificationFailed && !isWildcard) {
```

Comment:
Is this error only v1 or can it happen in v2 too?

Reply:
We still check that error, but we don't make it depend on the `isWildcard` variable